### ISSUE TITLE
Given your pretrained model, how can I evaluate it on IU dataset?

### ISSUE THREAD

**Issue by pierremaged** [cuhksz-nlp/R2GenCMN#3]

**Comment by zaaachos** [comment]
@pierremaged  First of all, you should comment out the code-block on the red rectangles in trainer.py. This will make the model run on evaluation mode
![image](https://user-images.githubusercontent.com/47610789/175814798-7b3dcd5a-c180-4b50-a12c-2d7d3b24d375.png)

Secondly, after downloading the pre-trained model, copy the path location of the downloaded model file, and type in the --resume argument a default value with the path location in string.
something like this:
`parser.add_argument('--resume', default='path\to\model\file', type=str, help='whether to resume the training from existing checkpoints.')`
You could find this in main.py file.
![image](https://user-images.githubusercontent.com/47610789/175814932-0aca5730-191a-482c-baf0-0e7e9345e2fc.png)
